src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  font-family: Roboto, sans-serif;
+  color: $main-text-color;
+  font-weight: 400;
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  padding: 32px 16px 16px;
+  background-color: #FFF;
+
+  &__img{
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+    margin-inline: auto;
+
+    img{
+      width: 100%;
+      height: 100%;
+    }
+  }
+
+  &__title {
+    margin: 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    color: $secondary-text-color;
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    @include display-flex;
+
+    margin-bottom: 24px;
+
+    &-item {
+      font-size: 10px;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    @include display-flex;
+
+    margin-bottom: 16px;
+    
+    &-text {
+      font-size: 12px;
+      color: $secondary-text-color;
+      line-height: 18px;
+    }
+
+    &-cost {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+    }
+  }
+
+  &__button {
+    box-sizing: border-box;
+    display: block;
+    width: 166px;
+    height: 40px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. use padding instead